mypy/stubgen.py

```diff
@@ -717,7 +717,7 @@ def get_init(self, lvalue: str, rvalue: Expression,
             typename = self.print_annotation(annotation)
             if (isinstance(annotation, UnboundType) and not annotation.args and
                     annotation.name == 'Final' and
-                    self.import_tracker.module_for.get('Final') in ('typing, typing_extensions')):
+                    self.import_tracker.module_for.get('Final') in ('typing', 'typing_extensions')):
```

Comment:
Nice that we were able to find this with the `__contains__` change.

Reply:
Yes, exactly!